### ISSUE TITLE
DJA0231 (VNCT-A) Remote assistance not working although enabled

### ISSUE THREAD

**Issue by gavindi** [Ansuel/tch-nginx-gui#767]
NB: Before submitting an issue, check if there is one already open that suits the problem you are having!

<b>If you are on the latest dev, please go to Cards >> System Extras >> Debug Report and press the Generate button.
  Please attach the file here:</b>

Device Model: DJA0231 (VCNT-A)
Gui Version: 9.3.77

Description of problem: Enabling remote assistance in the tile does noting. It shows that assistance is enabled but you can't access the router at the remote IP address.

How to reproduce (if possible): Enable it.

Media/Photos (if possible):
![rect941-1](https://user-images.githubusercontent.com/940596/62981896-e093b500-be6d-11e9-8b0f-78e4b6712b81.png)

**Comment by kevdagoat** [comment]
Be better off creating a support issue for the dja0231

**Comment by gavindi** [comment]
Sorry, I don't understand. I thought I was creating a support issue.....

**Comment by kevdagoat** [comment]
Hahaha. Me 5mins after getting up, sorry about the brevity.

Something like this:
https://github.com/Ansuel/tch-nginx-gui/issues/663
https://github.com/Ansuel/tch-nginx-gui/issues/723
https://github.com/Ansuel/tch-nginx-gui/issues/593
https://github.com/Ansuel/tch-nginx-gui/issues/627

**Comment by gavindi** [comment]
Ahhhh. Now I see. I will do.

**Comment by FrancYescO** [comment]
Is something shown when you open the modal? Remote assistance will not work on port 80/443 by wan

What's your Internet connection type?

Also /etc/config/network (remove sensitive credentials...) and a debug archive (system extra modal to generate) could be useful to debug the issue

**Comment by gavindi** [comment]
My connection type is ADSL2+

![Assistance](https://user-images.githubusercontent.com/940596/63121436-caa9ff80-bfe7-11e9-8cb1-4c1998d1d5c3.png)
[network.txt](https://github.com/Ansuel/tch-nginx-gui/files/3506948/network.txt)

**Comment by FrancYescO** [comment]
so at https://PUBLICIPADDRESS:11111, from another connection is still not working?

**Comment by gavindi** [comment]
Correct, even using the public address port 11111 it still doesn't work.

**Comment by kevdagoat** [comment]
Could we have a Logread?

Run: `logread-f`

Then enable remote access

**Comment by FrancYescO** [comment]
when setting random password to ON and enabling does a random password show up on save?

looks like there is a missing lib or something that will cause the error, yes probably a logread could help..

**Comment by gavindi** [comment]
Here's the logread:
Mon Aug 19 05:05:16 2019 daemon.info transformer[17606]: async run: /usr/share/transformer/scripts/user_reload.sh
Mon Aug 19 05:05:16 2019 daemon.info transformer[17606]: async run: /etc/fwdassist.sh
Mon Aug 19 05:05:16 2019 daemon.info transformer[17606]: async run: /usr/share/transformer/scripts/user_reload.sh
Mon Aug 19 05:05:16 2019 daemon.info transformer[17606]: async run: /etc/fwdassist.sh
Mon Aug 19 05:05:16 2019 daemon.info transformer[17606]: async run: /usr/share/transformer/scripts/user_reload.sh
Mon Aug 19 05:05:16 2019 daemon.info transformer[17606]: async run: /etc/fwdassist.sh

**Comment by FrancYescO** [comment]
wow nothing useful.. 
what about the password try? https://github.com/Ansuel/tch-nginx-gui/issues/767#issuecomment-522278382

does `curl 127.0.0.1:55555/ra` give empty result as reponse?

**Comment by gavindi** [comment]
1) No password shows up when you enable random password
2) Curl returns nothing

**Comment by FrancYescO** [comment]
any error shows up in logread when enabling random password?

**Comment by gavindi** [comment]
Correction. Setting a random password does actually create a password but I get error, connection refused when trying to access either by the specified port of 11111 or 8443

**Comment by Ansuel** [comment]
new on this?
